specs/rollup-node.md

```diff
@@ -92,40 +94,49 @@ Note that the sequencing windows overlap.
-        - `min_l2_timestamp = prev_l2_timestamp + l2_block_time`
-          - `prev_l2_timestamp` is the timestamp of the previous L2 block: the last block of the previous epoch,
-              or the L2 genesis block timestamp if there is no previous epoch.
-          - `l2_block_time` is a configurable parameter of the time between L2 blocks
-        - `max_l2_timestamp = max(l1_timestamp + max_sequencer_drift, min_l2_timestamp + l2_block_time)`
-          - `l1_timestamp` is the timestamp of the L1 block associated with the L2 block's epoch
-          - `max_sequencer_drift` is the most a sequencer is allowed to get ahead of L1
-      - The batch is the first batch with `batch.timestamp` in this sequencing window,
-        i.e. one batch per L2 block number.
-      - The batch only contains sequenced transactions, i.e. it must NOT contain any Deposit-type transactions.
+From the *first* block in the window only:
+
+- L1 block attributes:
+  - `blockNumber`
+  - `timestamp`
+  - `basefee`
+  - `random` (the output of the [`RANDOM` opcode][random])
+- Log entries emitted by the [Deposit Feed][[g-deposit-contract] contract, with the `TransactionDeposited` signature.
+- The `blockHeight` and `transactionIndex` of any transactions which emit `TransactionDeposited` logs (these values are
+  used to augmented the [deposited transactions][g-deposited] derived from them).
+
+From *each* block in the window:
+
+- Sequencer batches, derived from transactions that meet the following criteria:
+  - The transaction receiver (ie. `to`) is the [sequencer inbox] address.
+  - The transaction is signed (ie. `from`) by a recognized sequencer account.
+  - The calldata may contain a bundle of batches. *(calldata will be substituted with blob data in the future.)*
+
+- Batches must also meet the following filter conditions, or else will be ignored:
+  - `batch.epoch == sequencing_window.epoch`, i.e. for this sequencing window.
+  - `(batch.timestamp - genesis_l2_timestamp) % block_time == 0`, i.e. the timestamp matches with that of a valid l2
+    block.
+  - `min_l2_timestamp <= batch.timestamp < max_l2_timestamp`, i.e. the timestamp is within range. The parameters
+    are defined as follows:
+    - `min_l2_timestamp = prev_l2_timestamp + l2_block_time`, where:
+      - `prev_l2_timestamp` is the timestamp of the previous L2 block: the last block of the previous epoch,
+          or the L2 genesis block timestamp if there is no previous epoch.
+      - `l2_block_time` is a configurable parameter of the time between L2 blocks.
+    - `max_l2_timestamp = max(l1_timestamp + max_sequencer_drift, min_l2_timestamp + l2_block_time)`.
+      - `l1_timestamp` is the timestamp of the L1 block associated with the L2 block's epoch.
+      - `max_sequencer_drift` is the most a sequencer is allowed to get ahead of L1.
+  - The batch is the first batch with `batch.timestamp` in this sequencing window, i.e. one batch per L2 block number.
```

Comment:
s/one batch per L2 block number/only one batch per L2 block will be considered, and subsequent batches for the same L2 block will be ignored./